src/components/ProductList/index.jsx

```diff
@@ -1,23 +1,26 @@
 import React from 'react';
 import ItemList from 'components/ItemList';
-
+import { getProductStatus, sortProducts } from 'components/Utils/helpers';
 function ProductList({ products }) {
   return (
     <>
       {products && (
         <div>
-          {(products || []).map((product) => (
-            <div key={product.id}>
-              <ItemList
-                itemName={product.item}
-                docId={product.id}
-                nextPurchase={product.nextPurchase}
-                lastPurchasedDate={product.lastPurchasedDate || ''}
-                numberOfPurchases={product.numberOfPurchases}
-                estimatedDaysNextPurchase={product.estimatedDaysNextPurchase}
-              />
-            </div>
-          ))}
+          {(products || [])
+            .sort((a, b) => sortProducts(a, b))
```

Comment:
I have a curiosity, what is the sort criteria? Maybe by name or nextPurchase, I suggest you change the name of the function because it's a little confusing.